test/parallel/test-runner-cli.js

```diff
@@ -51,23 +51,19 @@ const testFixtures = fixtures.path('test-runner');
 
 {
   // User specified files that don't match the pattern are still run.
-  const args = ['--test', testFixtures, join(testFixtures, 'index.js')];
-  const child = spawnSync(process.execPath, args);
+  const args = ['--test', join(testFixtures, 'index.js')];
+  const child = spawnSync(process.execPath, args, { cwd: testFixtures });
```

Comment:
```suggestion
  const child = spawnSync(process.execPath, args, { __proto__: null, cwd: testFixtures });
```

Reply:
this is a test file, prototype pollution isn't really a concern here